.gitignore

```diff
@@ -1,131 +0,0 @@
-# VS Code
```

Comment:
I will let @tgsmith61591 comment on the actual implementation, but the `.gitignore` file will need to be added back before merging. It allows us to [specify files that we explicitly don't want to track](https://git-scm.com/docs/gitignore).

Reply:
It's my bad. I will undo the removal and add .gitignore.